elastic/kibana
# [Discover] Fix csv export with relative time filter from discover main view

## Summary

Fixes #122575

This PR redundant absolute filter from `csv export` generated from discover main view.

### Test notes
- Create an index pattern which contains data during the last N days.
- Choose Last 24h in the timepicker (note the URL contains only `(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:now-24h%2Fh,to:now))&_a=(columns:!(),filters:!(),index:'10881820-3b02-11eb-a646-7340c405390d',interval:auto,query:(language:kuery,query:''),sort:!(!(timestamp,desc)))`)
- Save the Saved Search - The URL still shows `(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:now-24h%2Fh,to:now))&_a=(columns:!(),filters:!(),grid:(),hideChart:!f,index:'10881820-3b02-11eb-a646-7340c405390d',interval:auto,query:(language:kuery,query:''),sort:!(!(timestamp,desc)))`
- Go on Share / CSV Export / Copy POST URL
- Decode URL
- Decoded URL should contain only one **relative (last 24 hours)** time range query, like that `query:(range:(timestamp:(format:strict_date_optional_time,gte:now-24h/h,lte:now))))`
